Podfile

```diff
@@ -52,11 +50,14 @@ end
 # Post Install
 #
 post_install do |installer|
-  # Remove min deploy target to clean up build warnings.
-  # See: https://stackoverflow.com/a/64048124
   installer.pods_project.targets.each do |target|
     target.build_configurations.each do |config|
+    	# Remove min deploy target to clean up build warnings.
+  		# See: https://stackoverflow.com/a/64048124
       config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
+      # Fix a code signing issue in Xcode 14 beta.
+  		# This solution is suggested here: https://github.com/CocoaPods/CocoaPods/issues/11402#issuecomment-1189861270
+      config.build_settings['CODE_SIGN_IDENTITY'] = ''
```

Comment:
Nitpick: The indentation here got mangled up. GitHub shows a mix of spaces and tabs:

<img width="224" alt="image" src="https://user-images.githubusercontent.com/1218433/200868231-a5359153-4ecc-40e9-89aa-fe37885730ce.png">

Reply:
Fixed here: https://github.com/Automattic/simplenote-ios/commit/0f4ac9a57b832b455a924def73fd70e17729795a